spec/interactors/add_playoffs_to_tournament_interactor_spec.rb

```diff
@@ -20,7 +20,7 @@
     @stages = create_list(:stage, 5)
   end
 
-  context 'ez lyfe' do
+  context 'PlayoffStageService mocked' do
```

Comment:
should probably be worded differently as well, but that applies to all spec-descriptions in this file, so that can be saved for later...

Reply:
well, but it is better than ez lyfe :thinking:

Reply:
better != good :thinking:

Reply:
But its still better :grin: